integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -4913,20 +4913,20 @@ msgid "Go to feedback"
 msgstr "Zum Feedback"
 
 #: cms/templates/dashboard/todo_dashboard_rows/_unreviewed_pages_row.html
-msgid "Pages that are waiting for review"
-msgstr "Seiten, die zum Review vorliegen"
+msgid "Pages pending approval"
+msgstr "Seiten, die zur Freigabe vorliegen"
```

Comment:
The German translation could stay as it is
```suggestion
msgid "Pages with pending approval"
msgstr "Seiten, die zur Freigabe vorliegen"
```
…or could be adapted like the following. Nothing really changes meaningfully, it's merely more of a literal translation of the changed English phrase, and some might argue that it would be more elegant to not have a comma here:
```suggestion
msgid "Pages with pending approval"
msgstr "Seiten mit ausstehender Freigabe"
```